README.md

```diff
@@ -137,7 +137,7 @@ wsa://com.apple.android.music
 | DuckDuckGo Privacy Browser+ | 5.142.2 | 12 | ✅
 | Easybell | 2.1.30 | 11 | ✅
 | EDS Lite | 2.0.0.237 | 12 | ✅ || Tested on an Intel x86-64 CPU (may work on AMD64 or ARM64). Recommended to add the exFAT module if you have a container that use this filesystem.
-| eGovPH | 1.3.8 | 13 | ⚠️ || Tested on a WSA install with GMS. Basic features work such as navigating through the app features/pages but for eReport, the app can't progress beyond the Current Location permission prompt as clicking the "Enable Location" doesn't do anything; the workaround is to access it via the Suggested eGovPH Services part. The PhilSys Digital ID page works as well. eKYC verification is problematic as the built-in laptop camera is wonky with eGovPH's image handling, affecting the verification process. Some pages display no content (notably FAQs and application details)
+| eGovPH | 1.9.1 | 13 | ⚠️ | Tested on a WSA install with GMS.  The app doesn't start properly if you didn't grant the permissions beforehand in android settings. For eReport, the app can't progress beyond the Current Location permission prompt as clicking the "Enable Location" doesn't do anything (even with the location permission granted for WSA in Windows); the workaround is to access it via the Suggested eGovPH Services part (found on the Home tab). eKYC verification is problematic as the built-in laptop camera is wonky with eGovPH's image handling, affecting the verification process (this also applies if you're registering an account from the same laptop for its face verification). Some pages display no content (notably FAQs and application details on some devices) |  Basic features work such as navigating through the app features/pages. The PhilSys Digital ID page works as well and able to recognize it in the verification site on my phone. Recommended to use an up-to-date version of Android System WebView since the app mostly relies on it.
```

Comment:
Thanks for extending my initial eGovPH commit. Mabuhay ka pare.

Reply:
It's a bit trickier to test on my WSA install since it's rooted (with Magisk), but it's doable for some extended testing for the newer version